apps/nextjs-website/src/components/atoms/Quote/Quote.tsx

```diff
@@ -0,0 +1,78 @@
+import { FormatQuote } from '@mui/icons-material';
+import { Box, Stack, Typography, useTheme } from '@mui/material';
+import Image from 'next/image';
+
+export type QuoteProps = {
+  readonly quote: string;
+  readonly color?: 'contrastText' | 'main' | 'light' | 'dark';
+  readonly backgroundImage?: {
+    readonly name: string;
+    readonly width: number;
+    readonly height: number;
+    readonly ext: string;
+    readonly mime: string;
+    readonly url: string;
+  };
+};
+
+const Quote = ({
+  quote,
+  color,
+  backgroundImage = {
+    name: 'hero-swiper.png',
+    width: 1920,
+    height: 1080,
+    ext: '.png',
+    mime: 'image/png',
+    url: '/images/hero-swiper.png',
+  },
+}: QuoteProps) => {
+  const { palette } = useTheme();
+  const textColor = color
+    ? palette.primary[color]
+    : palette.primary.contrastText;
+
+  return (
+    <Stack>
+      <Stack
+        height={{ xs: 468, sm: 560 }}
+        justifyContent={'start'}
+        alignItems={'center'}
+        direction={'row'}
+      >
+        <Stack
+          px={{ xs: 4, md: 18 }}
+          direction={'column'}
+          justifyContent={'start'}
+          zIndex={10}
+          sx={{ width: 'fit-content', maxWidth: '40%' }}
+        >
+          <FormatQuote sx={{ color: textColor, rotate: '180deg' }} />
+          <Typography color={textColor} variant='body1'>
```

Comment:
The `variant` is not corresponding to the design in Figma
![image](https://github.com/pagopa/developer-portal/assets/39835990/099ba376-3a40-4759-9125-c7c381fb37f2)